webrender_traits/src/units.rs

```diff
@@ -48,8 +51,10 @@ pub struct LayerPixel;
 
 pub type LayerRect = TypedRect<f32, LayerPixel>;
 pub type LayerPoint = TypedPoint2D<f32, LayerPixel>;
```

Comment:
Yeah, I don't mind either way.
We can make this a followup if you feel strongly about it. This will be an easy PR to write but will touch pretty much anything that isn't already in the diff so I'd rather not have it in this one.